### DISCUSSION TITLE
Learn about observational equality by McBride or cubical extension type (either cartesian or de morgan)

### DISCUSSION THREAD

**Discussion by xieyuheng** [cicada-lang/cicada-plct#110] [Ideas]
Learn about observational equality by McBride or cubical extension type (either cartesian or de morgan)

**Comment by xieyuheng** [comment]
https://github.com/cicada-lang/cicada/issues/43

**Comment by ice1000** [comment]
https://hal.inria.fr/hal-03367052v4/document This is a very good summary of the OTT research, with a predicative universe of strict proposition.
